mods/d2k/sequences/infantry.yaml

```diff
@@ -35,24 +43,24 @@ light_inf:
 		Facings: -8
 		Transpose: true
 	die1: DATA.R8
-		Frames: 382, 389, 396, 403, 410, 417, 424, 431, 438, 439, 440, 441
+		Frames: 382, 389, 396, 403, 410, 417, 424, 431, 438, 443, 448, 453
```

Comment:
This then makes me strongly suspect that all infantry will need similar fixes, because it wouldn't make sense for light infantry to do something special.  Can you please check this?